src/utils/hub-channel.js

```diff
@@ -97,10 +105,12 @@ export default class HubChannel {
   };
 
   updateScene = url => {
+    if (!this._permissions.update_hub) return "unauthorized";
```

Comment:
perms token expire in 5 minutes, so we'll need to add some plumbing to refetch a fresh one as needed

Reply:
we could potentially mitigate this issue for now by just making these tokens expire in 24 hours and revisiting, but it doesn't seem too hard to just refresh this (we'll just need another hub channel message to get a new token)

Reply:
yup, I'd imagined we'd need to request a new token before every kick/mute action anyway, so I'm going to leave it off to when I do that work